platforms/json_converter.py

```diff
@@ -0,0 +1,104 @@
+import requests
+import os
+import datetime
+
+''' Collects data from the main page. Since some fields may not always exist,
+try-except/if-else statements were included in order to prevent errors.
+'''
+def addHome (data, soup, url):
+    data[0]['ticket_url'] = '{}{}'.format(url, 'dates')
+    data[0]['name'] = soup.select('h1 span')[0].getText().strip()
+    ''' Getting the start and end dates is a bit more complicated and depends
+    on the page structure, thus there's another function for it
+    '''
+    getEventDates(data, soup)
+    try:
+        data[0]['location_name'] = soup.select('.venue span')[1].getText()
+    except Exception:
+        data[0]['location_name'] = ''
+    if len(soup.select('.all-dates-in-past')):
+        data[0]['state'] = 'Finished'
+    else:
+        data[0]['state'] = 'Unfinished'
+
+def getEventDates (data, soup):
+    # The event takes place over a single day
+    if len(soup.select('.mobile-only .date-range-date')) == 1:
+        start_date = end_date = soup.select('.mobile-only .date-range-date')[0].getText()
+        start_time = soup.select('.date-range-time-start')[0].getText()
```

Comment:
Instead of selecting the same element multiple times, you could just select it once and use the reference. Because each select is gonna cost you time.